src/Controller/Comment.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Masterclass\Controller;
+
+use Masterclass\Model\Comment as CommentModel;
+use Masterclass\Request;
+
+class Comment
+{
+    protected $commentModel;
+    protected $request;
+
+    public function __construct(CommentModel $comment, Request $request)
+    {
+        $this->commentModel = $comment;
+        $this->request = $request;
+    }
+
+    public function create()
+    {
+        if (!isset($_SESSION['AUTHENTICATED'])) {
+            // @todo: die, redirect, exit... what is desired functionality?
```

Comment:
I think that's what we call "a bug." Desired functionality should be to redirect.